ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -242,7 +242,9 @@ else
     export FEATURES="${FEATURES:-sriov performance sctp xt_u32 ovn metallb multinetworkpolicy vrf bondcni tuningcni}"
 fi
 export VALIDATIONS_FEATURES="${VALIDATIONS_FEATURES:-$FEATURES}"
-export TEST_RUN_FEATURES="${TEST_RUN_FEATURES:-$FEATURES}"
+if [[ -z "${TEST_RUN_FEATURES:-}" ]]; then
```

Comment:
~~~
export TEST_RUN_FEATURES="${TEST_RUN_FEATURES:-$FEATURES}"
~~~
Previous is preffered syntax in bash and functionally equal.